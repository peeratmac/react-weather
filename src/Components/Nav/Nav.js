import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import { NavLink } from 'react-router-dom';
import currentConditions from '../../images/currentConditions.svg';
import futureOutlook from '../../images/futureOutlook.svg';
import home from '../../images/home.svg';

export class Nav extends Component {
  render() {
    const linkStyle = {
      color: 'white',
      textDecoration: 'none'
    };

    return (
      <nav>
        <h2>Weather Station</h2>
        <ul className='nav-links'>
          <Link style={linkStyle} to='/current-conditions'>
            <li>
              Current Conditions
              <img src={currentConditions} />
            </li>
          </Link>
          <Link style={linkStyle} to='/future-outlook'>
            <li>
              Future Outlook <img src={futureOutlook} />
            </li>
          </Link>
          <Link style={linkStyle} to='/'>
            <li>
              Home <img src={home} />
            </li>
          </Link>
        </ul>
      </nav>
    );
  }
}

export default Nav;
