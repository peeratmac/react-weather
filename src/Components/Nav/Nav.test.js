import React from 'react';
import { shallow } from 'enzyme';
import { Nav } from './Nav';

describe('NAV', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav />);
  });

  it('should match the Nav Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
