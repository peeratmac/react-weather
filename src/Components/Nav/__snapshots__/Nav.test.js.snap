// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NAV should match the Nav Snapshot 1`] = `
<nav>
  <h2>
    Weather Station
  </h2>
  <ul
    className="nav-links"
  >
    <Link
      to="/current-conditions"
    >
      <li>
        Current Conditions
      </li>
    </Link>
    <Link
      to="/future-outlook"
    >
      <li>
        Future Outlook
      </li>
    </Link>
    <Link
      to="/"
    >
      <li>
        Home
      </li>
    </Link>
  </ul>
</nav>
`;
